### DISCUSSION TITLE
Fui aprobado en github education pero no he recibido los beneficios

### DISCUSSION THREAD

**Discussion by yostian07** [community/community#128884] [GitHub Education]
### Hi everyone,

Me registre hace mas o menos unos 5 dias para poder obtener los beneficios de github education me sale "aprobado tendras tus beneficios pronto"  pero no he recibido aun nada

**Comment by davevad93** [comment]
Hi @yostian07 

You still have to wait until your benefits are fully available, once they are, the message on the application page will state the following: 
`Congrats! Your request for @yostian07 was approved and your benefits are now available`
Keep in mind that the waiting time may vary from a few days to a few weeks.
